(1) Community Patch/Core Files/Core Changes/WorldChanges.sql

```diff
@@ -1,63 +1,39 @@
 -- This file changes the minimum city distance for major and minor civs depending on mapsize. Change these values as desired.
 
 -- DUEL - 3 Minimum for both
-
-UPDATE Worlds
-SET MinDistanceCities = '3'
-WHERE Type = 'WORLDSIZE_DUEL';
-
 UPDATE Worlds
-SET MinDistanceCityStates = '3'
+SET MinDistanceCities     = '3',
+    MinDistanceCityStates = '3'
 WHERE Type = 'WORLDSIZE_DUEL';
 
 -- Tiny - 3 Minimum for Major, 2 for Minor
-
-UPDATE Worlds
-SET MinDistanceCities = '3'
-WHERE Type = 'WORLDSIZE_TINY';
-
 UPDATE Worlds
-SET MinDistanceCityStates = '3'
+SET MinDistanceCities     = '3',
+    MinDistanceCityStates = '3'
 WHERE Type = 'WORLDSIZE_TINY';
 
 -- Small - 3 Minimum for Major, 2 for Minor
-
-UPDATE Worlds
-SET MinDistanceCities = '3'
-WHERE Type = 'WORLDSIZE_SMALL';
-
 UPDATE Worlds
-SET MinDistanceCityStates = '3'
+SET MinDistanceCities     = '3',
+    MinDistanceCityStates = '3'
 WHERE Type = 'WORLDSIZE_SMALL';
 
 -- Standard - 3 Minimum for Major, 3 for Minor
-
-UPDATE Worlds
-SET MinDistanceCities = '3'
-WHERE Type = 'WORLDSIZE_STANDARD';
-
 UPDATE Worlds
-SET MinDistanceCityStates = '3'
+SET MinDistanceCities     = '3',
+    MinDistanceCityStates = '3'
 WHERE Type = 'WORLDSIZE_STANDARD';
 
 -- Large - 4 Minimum for Major, 3 for Minor
```

Comment:
If anyone could help? Why comment says **4** update sets 3. And what is Major and Minor?

Reply:
This SQL statement values remained as they were in master, comment confused me so I've asked about it... I'm not sure what values here sure be.